cypress/tests/cli_test/BUILD

```diff
@@ -5,11 +5,15 @@ cypress_test(
     args = [
         "run",
         "--config-file=cypress.config.js",
+        "--browser=../../../../chrome/chrome"
     ],
     cypress = "//cypress/tests:node_modules/cypress",
     data = [
         "cli_test.cy.js",
         "cypress.config.js",
         "@aspect_rules_cypress//cypress/tests:node_modules",
     ],
+    browsers = [
```

Comment:
add a select for macos & linux with two different browser repositories so that this test passes on both?